src/EditorFeatures/TestUtilities2/Intellisense/TestState.vb

```diff
@@ -157,6 +157,12 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
             MyBase.SendTypeChars(typeChars, Sub(a, n, c) ExecuteTypeCharCommand(a, n, c))
         End Sub
 
+        Public Async Function SendTypeCharsAndWaitForUiRenderAsync(typeChars As String) As Task
+            Dim uiRender = WaitForUIRenderedAsync()
```

Comment:
Can this return a completed task before the SendTypeChars() happens? If so, is that bad?

Reply:
I guess if there's previous unprocessed completion related UI changes it's possible? But it should not happen in those tests